testing_and_setup/compass/ocean/drying_slope/marsh_flooding/100m/config_analysis.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<config case="analysis">
+  <add_link source="../comparison.py" dest="comparison.py"/>
```

Comment:
This can be a link to a more general location instead of it being hard-coded as done here.